src/rule.js

```diff
@@ -35,6 +36,10 @@ class Rule extends EventEmitter {
       this.setName(options.name)
     }
 
+    if (options && (options.id)) {
```

Comment:
I think we need the same non-numeric check here:
```suggestion
    if (options && (options.id && id !== 0)) {
```